persist-cli/src/main/java/io/ballerina/persist/pluralize/Pluralize.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.ballerina.persist.pluralize;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Transforms english words from singular to plural form.
+ */
+public class Pluralize {
+
+    public static String getResourceNameInPlural(String resourceName) {
```

Comment:
```suggestion
    public static String plural(String word) {
```

Reply:
This is a generic API, not bind to resource name

Reply:
Shouldn't it be pluralize?

Reply:
I referred the name from https://github.com/plurals/pluralize

Reply:
Ah I was thinking on the point of coding standards. As a function shouldnt it be a verb? `plural` is either adj/noun. 🤔

Reply:
Changed the class name to `Pluralizer ` and the method name to `pluralize`